packages/itwinui-css/src/global.scss

```diff
@@ -16,7 +16,7 @@
 :where(.iui-root) {
   color: var(--iui-color-text);
   font-size: var(--iui-font-size-1);
-  line-height: var(--iui-size-l);
+  line-height: 1.5;
```

Comment:
If I understand the change correctly, all `line-height` for specific components are removed and they now all inherit from this global value off root? Also, what is unitless? Is that implied to be pixel? And shouldn't we still have a variable for this too?

Reply:
Here's an article on it which will explain better than I can: https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#prefer_unitless_numbers_for_line-height_values

My basic explanation is the line height will adjust based on the font size.  12px text will have small line height, 32px text will have larger small height.